src/java/org/apache/cassandra/config/Config.java

```diff
@@ -322,6 +322,8 @@
 
     public boolean enable_materialized_views = true;
 
+    public volatile boolean enable_drop_compact_storage = true;
```

Comment:
Does this one need to be volatile?

Reply:
We have a set method in {{DatabaseDescriptor}} used by the test. So potentially, the variable can be read and written by 2 different threads. I prefered to be careful.